errors/src/parser/parser_errors.rs

```diff
@@ -80,6 +80,32 @@ create_errors!(
         help: None,
     }
 
+    /*
+    /// When a member const comes after a member variable.
+    @formatted
+    member_const_after_var {
+        args: (),
+        msg: "Member variables must come after member consts.",
+        help: None,
+    }
+
+    /// When a member const comes after a member function.
+    @formatted
+    member_const_after_fun {
+        args: (),
+        msg: "Member functions must come after member consts.",
+        help: None,
+    }
+
+    /// When a member variable comes after a member function.
+    @formatted
+    member_var_after_fun {
+        args: (),
+        msg: "Member functions must come after member variables.",
+        help: None,
+    }
```

Comment:
If I readd these, weird hisenbugs seem to happen. Suddenly the columns of unrelated errors change in the expectations.

Reply:
It should be fine if only the error codes change in the expectations (e.g `EPAR0370009`).
This would happen because the `create_errors!` macro generates codes in order that error codes are defined in the file.

Reply:
![Facepalm](https://upload.wikimedia.org/wikipedia/commons/3/3b/Paris_Tuileries_Garden_Facepalm_statue.jpg)